core/src/main/java/org/kiwix/kiwixmobile/core/utils/KiwixDialog.kt

```diff
@@ -91,7 +91,14 @@ sealed class KiwixDialog(
     R.string.hotspot_dialog_title,
     R.string.hotspot_dialog_message,
     R.string.go_to_settings,
+    R.string.go_to_wifi_settings_label
+  )
+
+  object WifiOn : KiwixDialog(
```

Comment:
This name could use a bit more. `WifiOn` doesn't tell me anywhere near as much as it's sibling `StartHotspotManually`